src/lib/dom-api-event.html

```diff
@@ -71,10 +71,18 @@
       get path() {
         if (!this.event._path) {
           var path = [];
-          var o = this.rootTarget;
-          while (o) {
-            path.push(o);
-            o = Polymer.dom(o).parentNode || o.host;
+          var current = this.rootTarget;
+          while (current) {
+            path.push(current);
+            var insertionPoints = Polymer.dom(current).getDestinationInsertionPoints();
+            if (insertionPoints.length) {
+              for (var i = 0; i < insertionPoints.length - 1; i++) {
+                path.push(insertionPoints[i]);
+              }
+              current = insertionPoints[insertionPoints.length - 1];
+            } else {
+              current = Polymer.dom(current).parentNode || current.host;

```

Comment:
Don't think that makes sense. Destination insertion points and parentNode are not related.